MPF.Core/Modules/XboxBackupCreator/Parameters.cs

```diff
@@ -0,0 +1,684 @@
+            {
+                info.Artifacts ??= [];
+
+                if (File.Exists(logPath))
+                    info.Artifacts["log"] = GetBase64(GetFullFile(logPath!)) ?? string.Empty;
+                if (File.Exists($"{basePath}.dvd"))
+                    info.Artifacts["dvd"] = GetBase64(GetFullFile($"{basePath}.dvd")) ?? string.Empty;
+                //if (File.Exists($"{baseDir}DMI.bin"))
+                //    info.Artifacts["dmi"] = Convert.ToBase64String(File.ReadAllBytes($"{baseDir}DMI.bin")) ?? string.Empty;
+                // TODO: Include PFI artifact only if the hash doesn't match known PFI hashes
+                //if (File.Exists($"{baseDir}PFI.bin"))
+                //    info.Artifacts["pfi"] = Convert.ToBase64String(File.ReadAllBytes($"{baseDir}PFI.bin")) ?? string.Empty;
+                //if (File.Exists($"{baseDir}SS.bin"))
+                //    info.Artifacts["ss"] = Convert.ToBase64String(File.ReadAllBytes($"{baseDir}SS.bin")) ?? string.Empty;
+                //if (File.Exists($"{baseDir}RawSS.bin"))
+                //    info.Artifacts["rawss"] = Convert.ToBase64String(File.ReadAllBytes($"{baseDir}RawSS.bin")) ?? string.Empty;
+            }
+        }
+
+        /// <inheritdoc/>
+        public override List<string> GetLogFilePaths(string basePath)
+        {
+            var logFiles = new List<string>();
+            string baseDir = Path.GetDirectoryName(basePath) + Path.DirectorySeparatorChar;
+            switch (this.Type)
+            {
+                case MediaType.DVD:
+                    string? logPath = GetLogName(baseDir);
+                    if (!string.IsNullOrEmpty(logPath))
+                        logFiles.Add(logPath!);
+                    if (File.Exists($"{basePath}.dvd"))
+                        logFiles.Add($"{basePath}.dvd");
+                    if (File.Exists($"{baseDir}DMI.bin"))
+                        logFiles.Add($"{baseDir}DMI.bin");
+                    if (File.Exists($"{baseDir}PFI.bin"))
+                        logFiles.Add($"{baseDir}PFI.bin");
+                    if (File.Exists($"{baseDir}SS.bin"))
+                        logFiles.Add($"{baseDir}SS.bin");
+                    if (File.Exists($"{baseDir}RawSS.bin"))
+                        logFiles.Add($"{baseDir}RawSS.bin");
+
+                    break;
+            }
+
+            return logFiles;
+        }
+
+        #endregion
+
+        #region Information Extraction Methods
+
+        /// <summary>
+        /// Determines the file path of the XBC log
+        /// </summary>
+        /// <param name="baseDir">Base directory to search in</param>
+        /// <returns>Log path if found, null otherwise</returns>
+        private static string? GetLogName(string baseDir)
+        {
+            if (IsSuccessfulLog($"{baseDir}Log.txt"))
+                return $"{baseDir}Log.txt";
+
+            if (IsSuccessfulLog($"{baseDir}log.txt"))
+                return $"{baseDir}log.txt";
```

Comment:
This may or may not work the way you're expecting. If you can get someone on Linux to test this, that would be good, especially since Windows is case-insensitive by default.

Reply:
I can remove the second one as I'm enumerating all txt files right after that anyway. I've just noticed it seems common enough that people have `log` instead of `Log`

Reply:
I think that keeping both is fine, I just wanted to see what Linux users experience with this, given the difference in case-sensitivity. I've seen this issue crop up before in other programs.

Reply:
It was redundant anyway, no harm done